source/docs/01-introduction.xml

```diff
@@ -233,17 +233,38 @@
                      The release technically removes the &lt;fingerprint&gt; element, which has been 
                      deprecated for ten years. It also removes the elements &lt;pgHead2&gt; and 
                      &lt;pgFoot2&gt;, which are now superseded by the <att>func</att> attribute on 
-                     <gi scheme="MEI">pgHead</gi> and <gi scheme="MEI">pgFoot</gi> respectively. Most other changes affect more specific aspects in the model of MEI, usually expressed in
-                     attributes. These can be traced in the detailed Release Notes auto-generated from 
-                     the Pull Requests on GitHub. A larger group of changes affects the internal class
-                     structure of MEI only, where significant effort went into improved consistency in
-                     naming things. While this set of changes does not affect end users of MEI during
-                     validation of their files, they may have consequences for local customizations which
-                     reference classes not available anymore. If you have advanced local customizations
-                     based on MEI v4 or older releases, please check that the rules provided still work as
-                     expected under v5. A very helpful addition for this task may be the validation for MEI
-                     customizations, which is now available and used for all customizations officially
-                     provided by MEI.</p>
+                     <gi scheme="MEI">pgHead</gi> and <gi scheme="MEI">pgFoot</gi> respectively. 
+                     Most other changes affect more specific aspects in the model of MEI, usually 
+                     expressed in attributes. These include the refinement of the encoding of key 
+                     signatures, with @key.sig moved to <att>keysig</att>, @keysig.show moved to 
+                     <att>keysig.visible</att>, and @keysig.showchange and @sig.showchange moved to 
```

Comment:
There is probably an XSLT-reason why the deprecated attributes are not wrapped in `<att>…</att>`?

Reply:
Yes, these are the removed attributes. I was wrong about these being linked to the current guidelines (which wouldn't work), but this only happens for the elements. You are right, I will add these att tags.